src/js/common/schemaform/FormIntroButtons.jsx

```diff
@@ -22,12 +22,18 @@ class FormIntroButtons extends React.Component {
     this.props.router.push(this.props.route.pageList[1].path);
   }
 
-  handleLoadForm = () => {
+  handleLoadPrefill = () => {
+    if (this.props.loggedIn) {
+      this.handleLoadForm(true);
+    } else {
+      this.goToBeginning();
+    }
+  }
+
+  handleLoadForm = (prefill = false) => {
     // If successful, this will set form.loadedData.metadata.returnUrl and will
     //  trickle down to this.props to be caught in componentWillReceiveProps
-    this.props.fetchInProgressForm(this.props.formId, this.props.migrations)
-      // Should probably do GA or sentry or something, but for now...
-      .catch((e) => console.error('Error loading form:', e)); // eslint-disable-line no-console
```

Comment:
I think I added this when I was testing the loading screen errors. Gave me an `unhandled rejection` error (or some such) without it, so I don't know if we want to get rid of this...unless I'm missing something.